notes_app/notes/api.py

```diff
@@ -39,6 +39,44 @@ def delete_detail(self, object_list, bundle):
         raise Unauthorized("Deletes not allowed")
 
 
+# Custom User Authorization
+class UserObjectsOnlyAuthorization(Authorization):
```

Comment:
it's better to have some unified name structure for authorization classes. `<base_name_of_resource>Authorization` for example. if resource class name is `UserResource` then here should be `UserAuthorization`. For additional info you have comments or docstrings